_articles/disaster-recovery.md

```diff
@@ -57,3 +57,7 @@ Once we have our data backed up we can reinstall the OS if we are ready to do th
 
 ![Restoring data in Deja-Dup](/images/backup/deja_overview.png)
 
+## Future Steps
+
+Now that we have our data and a clean install of the OS let's prepare so that we will be ready if this issue happens again using our backup article [here](/articles/backup-files).
```

Comment:
Sentence adjustments:

Now that we have our data, and a clean install of the OS, let's prepare for if this issue happens again. It's good practice to set up scheduled backups. Our backup article can help: